niceman/retrace/tests/test_packagemanagers.py

```diff
@@ -13,6 +13,7 @@
 
 def test_identify_packages():
     files = ["/usr/share/doc/xterm/copyright",
+             "/usr/games/alienblaster",
```

Comment:
so could the test now become an actual basic test (verify that if file is present -- package should be identified correctly etc), may be without even mocking (i.e. debian systems specific), not just a smoke run with prints? ;)